rules/no-implicit-end.js

```diff
@@ -32,6 +44,14 @@ module.exports = function() {
       return false;
     }
 
+    if (is(node, 'bpmn:BoundaryEvent') && isCompensationEventDefinition(node)) {
```

Comment:
We still want to treat this as an implicit end like any other boundary event.

![image](https://github.com/user-attachments/assets/47f00ead-9b1f-4913-b6dc-dccd3a15a156)

For comparison, timer boundary event with the implicit end error.

![image](https://github.com/user-attachments/assets/e6426f89-6587-44e8-bfa8-6b403dc92ce6)